libs/pika/topology/src/topology.cpp

```diff
@@ -1579,34 +1576,32 @@ namespace pika { namespace threads {
     }
 
     ///////////////////////////////////////////////////////////////////////////
-    topology& create_topology()
+    topology& create_topology() noexcept
```

Comment:
This shouldn't be noexcept because the `topology` constructor isn't noexcept (and it shouldn't be noexcept because it can throw an exception right there in the constructor body).